src/components/Landing_Page/hero.jsx

```diff
@@ -1,5 +1,5 @@
 import { A } from "@solidjs/router";
-import { setOS } from "../docpage/install";
+import { setOS, setOSCMD, docker_cmd } from "../docpage/install";
```

Comment:
why is it needed? clicking on the docker btn in home page gets to the right docker section in install docs